tools/mock-service-host/src/mid/coordinator.ts

```diff
@@ -291,7 +291,11 @@ export class Coordinator {
                 }
 
                 // simplified LRO
-                ret = replacePropertyValue('provisioningState', 'Succeeded', ret)
+                ret = replacePropertyValue(
+                    'provisioningState',
+                    'Succeeded',
+                    replacePropertyValue('nextLink', null, ret)
```

Comment:
consider move line 297 to paging handling area (currently line 288~291) to make logic more clear:

~~~
          if (!Array.isArray(ret)) {
                    // simplified paging
                    ret = lodash.omit(ret, 'nextLink')
            }
           else {
                  ret = replacePropertyValue('nextLink', null, ret)
            }
~~~